keps/sig-etcd/4578-server-feature-gate/kep.yaml

```diff
@@ -0,0 +1,19 @@
+title: Feature Gate in etcd
+kep-number: 4578
+authors:
+  - "@siyuanfoundation"
+  - "@stackbaek"
+owning-sig: sig-etcd
+status: provisional
+creation-date: yyyy-mm-dd
+reviewers:
+  - "@ahrtr"
+  - "@serathius"
+  - "@jmhbnz"
+  - "@wenjiaswe"
+  - "@fuweid"
+  - "@logicalhan"
```

Comment:
That's a lot of reviewers, no need to have so many people, but if you want to keep it please make sure that you got review from everyone listed.

Reply:
ok. I'll try to get their reviews.